tmt/cli.py

```diff
@@ -832,3 +832,40 @@ def init(context, path, template, force, **kwargs):
         tmt.Test.create('/tests/example', 'shell', tree, force)
         tmt.Plan.create('/plans/example', 'full', tree, force)
         tmt.Story.create('/stories/example', 'full', tree, force)
+
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#  Status
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+@main.command()
+@click.pass_context
+@click.argument('path', default=tmt.utils.WORKDIR_ROOT)
+@click.option(
+    '-i', '--id', 'id_', help='Run id (name or directory path) to show status of.', metavar="ID")
+@click.option(
+    '--abandoned', is_flag=True, default=False,
+    help='List runs with active provision.')
+@click.option(
+    '--active', is_flag=True, default=False,
+    help='List runs where at least one step has not been finished.')
+@click.option(
+    '--finished', is_flag=True, default=False,
+    help='List all finished runs.')
+@verbose_debug_quiet
+def status(context, id_, path, abandoned, active, finished, **kwargs):
```

Comment:
Hint: If you use `Status._save_context()` you don't have to pass individual options and can easily check them using `self.opt()` later.